internal/storage/v2/grpc/trace_storage.proto

```diff
@@ -62,6 +63,46 @@ message GetOperationsResponse {
   repeated Operation operations = 1;
 }
 
+message TraceQueryParams {
```

Comment:
It would be good to synchronize all the semantic descriptions across different API into one place. We have api_v3, storage/v2 (Go interfaces), and this proto.

Can we just refer to `api_v3.query_service.proto`? I.e. add a comment to the type "the meaning of all fields is the same as in {link}"

Reply:
@yurishkuro the IDL of api_v3 is slightly different than this IDL which makes me wonder if we should keep them separate? If we want them to be the same, then I wonder if they should also re-use the same messages?

Reply:
We want to keep them separate because they represent different layers with slightly different needs

Reply:
Does it make sense to keep the descriptions separate as well? I think it might things easier/less confusing for users implementing this interface to have to go to another interface to find out what these fields mean.

Reply:
I would rather point out differences if some appear in the future, but not duplicate the semantic explanation of the API